std/asyncio/system/SystemGroup.hx

```diff
@@ -0,0 +1,39 @@
+package asyncio.system;
+
+import haxe.NoData;
+import haxe.Callback;
+import haxe.errors.NotImplemented;
+
+/**
+	Represents a users group registered in OS.
+
+	TODO:
+	Not sure what would be the best underlying type for cross-platform, hence `@:coreType`
```

Comment:
I think both users and groups have `Int` ID numbers, even under Windows, but I could be wrong. I base this on libuv using `int` for `chmod`.

Reply:
I'm not sure.
https://en.wikipedia.org/wiki/Security_Identifier

Reply:
Yeah also https://github.com/libuv/libuv/blob/v1.x/src/win/fs.c#L1712-L1713 …